subtree-splits/blt-project/composer.json

```diff
@@ -8,14 +8,14 @@
         "acquia/blt": "10.x-dev",
         "acquia/drupal-spec-tool": "*",
         "acquia/lightning": "^4.0.0",
+        "acquia/memcache-settings": "*",
```

Comment:
I see this module includes 2.0, not 2.0-alpha7. Is that a problem? There was a discussion on another PR I remember.

Reply:
Yep 2.0 is the version that we now publicly recommend, and that works with the recommended memcache settings.